src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/ReferralConcluder.kt

```diff
@@ -88,4 +88,10 @@ class ReferralConcluder(
       actionPlanRepository.countNumberOfAttemptedSessions(referral.id)
     } ?: 0
   }
+
+  private fun countSessionsAttended(referral: Referral): Int {
+    return referral.currentActionPlan?.let {
+      actionPlanRepository.countNumberOfAttendedSessions(referral.id)
```

Comment:
Do we need the `currentActionPlan`?  It looks like `actionPlanRepository.countNumberOfAttendedSessions`  returns 0 if there's no approved action plan anyway; is that correct?

Reply:
Hmm the ticket didn't mention anything to do with the current action plan. I'm not sure, I just implemented it the way it was but only checking for an actual attendance of an appointment.

Reply:
The appointments would be moved from any previous action plans anyway wouldn't they?